yarn.lock

```diff
@@ -1585,21 +1619,31 @@ doctrine@^3.0.0:
   dependencies:
     esutils "^2.0.2"
 
+eastasianwidth@^0.2.0:
```

Comment:
"tsup#sucrase#glob#jackspeak#@isaacs#cliui#string-width" depends on it
   - Hoisted from "tsup#sucrase#glob#jackspeak#@isaacs#cliui#string-width#eastasianwidth"
   - Hoisted from "tsup#sucrase#glob#jackspeak#@isaacs#cliui#wrap-ansi#string-width#eastasianwidth"